src/main/java/databaseInteracting/DataBaseConnection.java

```diff
@@ -0,0 +1,27 @@
+package databaseInteracting;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnection {
+  private static String url;
+  private static String user;
+  private static String password;
+
+  /**
+   * Method constructor of the class to receive the parameters of database connection.
+   * @param url String - URL of database.
+   * @param user String - User name of database.
+   * @param password String - Password of database.
+   */
+  public DataBaseConnection(String url, String user, String password) {
+    DataBaseConnection.url = url;
+    DataBaseConnection.password = password;
+    DataBaseConnection.user = user;
+  }
```

Comment:
Not a good practice to initialize static variable in a constructor. Each time you create a new object it will overide all the values